kuma/landing/jinja2/landing/homepage.html

```diff
@@ -142,7 +142,7 @@ <h2>{% include 'includes/icons/general/smile.svg' %} {{ _('Help improve MDN') }}
 {% block contrib_popup %}
     {%- if contribution_popup %}
         {%- with form=contribution_form, is_popover=True %}
-            {%- include "contributions/includes/contribution-form.html" %}
+            {%- include "payments/includes/payments-form.html" %}
```

Comment:
Another change is needed on line 156 to load the correct JavaScript.

Reply:
Good spot! I've addressed this.